Submissions/@Aakibvah.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "MahmmadAakib Vahora",
+  "img": "https://avatars.githubusercontent.com/u/87711863?v=4",
+  "email": "aakibv290@gmail.com",
+  "links": {
+    "website": "https://aakibvah.github.io/",
+    "linkedin": "https://www.linkedin.com/in/mahmmadaakibvahora",
+    "github": "https://github.com/Aakibvah"
+  },
+  "jobTitle": "Web Developer [React and Node]",
```

Comment:
I also don't think the brackets here will be an issue, but the README says not to include them - perhaps use () instead?